src/cellular/dataCollection.cpp

```diff
@@ -10,13 +10,46 @@
 
 typedef struct Ensemble10_eventData_
 {
+    /**
+     * @brief Average temperature value of the water in Celsius provided
+     * by the temperature sensor
+     */
     double temperature;
+    /**
+     * @brief Truncated average of whether or not system in water over
+     * accumulation period
+     *
+     */
     int32_t water;
+    /**
+     * @brief Array saving average of the accumulated accelerometer data on the
+     * x, y, and z axis in g scaled up by 16834
```

Comment:
Does this mean that if `acc` has the value `{1, 0, 0}`, I should interpret this as [0.000059, 0, 0] g's of acceleration?

Conversely, should I expect [1, 0, 0] g's of acceleration to be represented at `{16834, 0, 0}`?

Reply:
when you specify this, please add units for the real-world value so that it is unambiguously the real-world value vs the stored representation.

Reply:
This ends up being confusing - are we stating that the values as stored in memory are directly interpretable as G's of acceleration?

Additionally, in line 26, I think you have the wrong constant.

One suggested wording might be this:
```
acc[i] = acceleration[i] * 16384, where `acceleration` is the acceleration in G's (multiple of Earth gravity), and acc is the stored representation, stored as `int32_t`.  Thus, acc[i] = 16384 = 1 G.
```

Consider updating the other field documentation to be similar.